services/web/server/src/simcore_service_webserver/exporter/formatters/formatter_v1.py

```diff
@@ -287,6 +350,10 @@ async def import_files_and_validate_project(
     project_uuid = str(project.uuid)
 
     try:
+        await _fix_node_run_hashes_based_on_old_project(
+            project, project_file, shuffled_data
+        )
+        await _fix_file_e_tags(project, links_to_new_e_tags)
```

Comment:
I see you generate new node states and also new tags

Reply:
I actually fix the run hash if the old one was still correct